### PR TITLE
[Win] Improve design of the scrollbars

### PR DESCRIPTION
**Pull request by stefanivic**
Hyper on Windows uses default scrollbars that do not go well with the new design for Windows. 
I updated the design to change only on win32 so that MacOS and Linux remain the same.

Would love to get some feedback on this (code or design).

### CODE REVIEW

File: lib/components/term.js [hunk 1]
```diff
@@ -184,13 +184,35 @@ export default class Term extends Component {
...
-      }
-      ${css}
-    `], {type: 'text/css'});
-    return URL.createObjectURL(blob);
+    if(process.platform === 'win32') {
+      const scroll = new Blob([`
+        .cursor-node[focus="false"] {
+          border-width: 1px !important;
+        }
+        ::-webkit-scrollbar {
```
Comment (fearphage) [inline lib/components/term.js]: I have a suggested simplifcation.

```js
getStylesheet(css) {
  const osSpecifcCss = process.platform === 'win32' 
    ? `/* win32 css things */`
    : ''
  ;

  return URL.createObjectURL(new Blob([`
    .cursor-node[focus="false"] {
       border-width: 1px !important;
    }
    ${osSpecificCss}
    ${css}
  `], {type: 'text/css'}));
} 
```

It's less repetitive and it makes it easy to slot in the os-specific bits.

Comment (stefanivic) [inline lib/components/term.js]: Yeah, this is great. Will update. 
Thanks.

**Comment by matheuss** [comment]
Amazing @stefanivic ☺️ 
I think applying white to it is not a good idea – it makes more sense to apply something like the `borderColor`. Given that, I rebased it for you and changed the implementation to use the border color 😄 Let me know what you think 👌 

kapture 2016-11-19 at 14 30 03

**Comment by stefanivic** [comment]
@matheuss yeah, looks even better. :D

**Comment by matheuss** [comment]
Thank you @stefanivic 🙌

**Pull request merged**